docs/sources/tutorials/get-started.md

```diff
@@ -71,17 +91,167 @@ services:
+    path_targets = [{"__path__" = "/var/log/*.log"}]
+    sync_period = "5s"
+}
+```
+
+This component creates a [local.file_match][] component named `local_files` with an attribute that tells {{< param "PRODUCT_NAME" >}} which files to source, and to check every 5 seconds.
+
+### Second Component: Scraping
+
+Paste this component next in the `config.alloy` file:
+
+```alloy
+loki.source.file "log_scrape" {
+    targets    = local.file_match.local_files.targets
+    forward_to = [loki.write.grafana_loki.receiver]
+    tail_from_end = true
+}
+```
+
+This configuration creates a [loki.source.file][] component named `log_scrape`, and shows the pipeline concept of {{< param "PRODUCT_NAME" >}} in action. The `log.scrape` component does the following:
+
+1. It applies to the `local_files` component (its "source" or target).
+1. It forwards the logs it scrapes to the "receiver" of another component called `grafana_loki` that you will define next.
+1. It provides extra attributes and options, in this case, you will tail log files from the end and not ingest the entire past history.
+
+### Third Component: Write Logs to Loki
+
+Paste this component last in your configuration file:
+
+```alloy
+loki.write "grafana_loki" {
+  endpoint {
+    url = "http://localhost:3100/loki/api/v1/push"
+
+    // basic_auth {
+    //  username = "admin"
+    //  password = "admin"
+    // }
+  }
+}
+```
+
+This last component creates a [loki.write][] component named `grafana_loki` that points to `http://localhost:3100/loki/api/v1/push`.
+This completes the simple configuration pipeline.
+
+{{< admonition type="tip" >}}
+The `basic_auth` is commented out because the local `docker compose` stack doesn't require it. 
+It is included in this example to show how you can configure authorization for other environments.
+For further authorization options, refer to the [loki.write][] component reference.
+
+[loki.write]: ../../reference/components/loki.write/
+{{< /admonition >}}
+
+This connects directly to the Loki instance running via `docker compose` from the earlier step.
+
+## Reload the Configuration
+
+Copy your local `config.alloy` file into the default configuration file location, which depends on your OS.
+
+{{< code >}}
+
+```macos
+sudo cp config.alloy $(brew --prefix)/etc/alloy/config.alloy
+```
+
+```linux
+sudo cp config.alloy /etc/alloy/config.alloy
+```
+
+{{< /code >}}
+
+Finally, call the reload endpoint to alert {{< param "PRODUCT_NAME" >}} to the configuration change without the need
+for restarting the system service.
+
+```bash
+    curl -X POST http://localhost:12345/-/reload
+```
+
+{{< admonition type="tip" >}}
+If this step does not work for you, please note that in the install instructions, enabling it requires
+one extra optional step for Linux, while this is enabled by default on MacOS.
+{{< /admonition >}}
+
+The alternative to using this endpoint is to reload the {{< param "PRODUCT_NAME" >}} configuration, which can
+be done as follows:
```

Comment:
Do we want to suggest this?  If yes, then can we simply link to the relevant docs?  My concern here is that someone not reading carefully (and users don't always read all the words) will reload the config and then restart the service. This might be better left out. This could be considered out of scope for the tutorial (we don't need to cover all possibilities.. just what's needed to follow through on things and get the desired result).
```